src/components/ButtonLink.tsx

```diff
@@ -2,19 +2,31 @@ import { useNavigate } from "react-router-dom";
 import { Button, ButtonProps } from "./Button";
 
 type Props = Omit<ButtonProps, "onClick"> & {
```

Comment:
tohle urcite ne, to je spatne. Co kdyz tohle tlcitko budes chtit pouzit nekde jinde nez u te lokace?